addons/survey/static/src/js/survey_form.js

```diff
@@ -430,9 +433,31 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
         self.$(selectorsToFadeout.join(',')).fadeOut(this.fadeInOutDelay, function () {
             resolveFadeOut();
         });
+        // Background management - Will the background change between current and next/previous questions ?
+        var refreshBackground = self.options.reloadBackgroundOnNext || options.goBack;
+        if (refreshBackground) {
+            // If we make a transition, hide previous background by setting 'background overlay' opacity = 1
+            $('div#wrapwrap').css("box-shadow", "inset 0 0 0 10000px rgba(255,255,255,1)");
+        }
 
         Promise.all([fadeOutPromise, nextScreenPromise]).then(function (results) {
-            return self._onNextScreenDone(results[1], options);
+            var refreshBackgroundPromise = Promise.resolve(false);
+            var next_question_background = results[1].background_image_url;
+            self.options.reloadBackgroundOnNext = results[1].reload_background_on_next;
```

Comment:
Maybe we could name "results[1]" since it's now being used more? Something like "nextScreenResult" ?